### DISCUSSION TITLE
no wifi connection established

### DISCUSSION THREAD

**Discussion by CSH12** [beep-projects/SystaPi#16] [Q&A]
Hi, 
I have seen the project and wanted to give it a try - but i'm not that well trained in programming :-( 
I hopefully did everthing you explained in the readme, but unfortunately my wifi connection couldn't get established. 

I have taken an rpi 4 that I had left over from another project and try to get it running with the scripts, but within the secondrun.log is stated that the rpi is "waiting for internet access ..." - that's it. 
If I connect the eth0 to the switch, the scripts are running and the downloads are running, but as I understood this is not what has to be done for to configure the rpi correctly.

For to make it a little easier I took over your User and PW so that I just used my wifi and wpa to get it running. 
Do I have to set the wifi or wpa in some quotes if there are some symbols in it? 

here the logs of the first and second run.
[firstrun.log](https://github.com/beep-projects/SystaPi/files/10290901/firstrun.log)
[secondrun.log](https://github.com/beep-projects/SystaPi/files/10290902/secondrun.log)

I would be glad if you could assist a little.
Thanks

Chris

edit: 

**Now I have hacked the scripts a litte so that it was running and it is working now - after round about 100 runs ;-)**

**Comment by beep-projects** [comment]
Hey Chris,

glad to read that it now works for you. Can you share what you had to tweak to get it running for you?

From your above, eth0 should be connected to the Systa Comfort and the WiFi should be used for connecting to the local network.
If you connect eth0 to your network, I am not sure what happens. The dnsmasq on the SystaPi might assign adresses to your other units, breaking their connectivity.

Beep
